packages/runtime/container-runtime/src/deltaScheduler.ts

```diff
@@ -30,7 +30,7 @@ export class DeltaScheduler {
 	public static readonly processingTime = 50;
 
 	// The increase in time for processing ops after each turn.
```

Comment:
Switch to `/** this style */` comment so you can see the comment on hover (consider updating the other fields too)